### PR TITLE
Add The "Contact Me" Section

### PR DESCRIPTION
**Pull request by calebchris000**
The last part of the portfolio was the Contact Me section. This was implemented at the end of the portfolio.

### CODE REVIEW

**Review by amejid — COMMENTED** [review]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, with a README file that does not follow the README template provided by Microverse. Kindly update your README file using [this template](https://github.com/microverseinc/readme-template).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by calebchris000** [comment]
Hi @amejid ,
I've reviewed my README, but I don't see any issue related to your request. Can you please point out where the issue is coming from?

**Comment by amejid** [comment]
Hi @calebchris000, I am not seeing the table of contents, "Key Features" section. All relevant sections of the README should be present. The only optional section is the "FAQ" section. Please update it accordingly.

**Comment by calebchris000** [comment]
Hi @amejid , my previous code reviewer told me that section was to be removed because 'it is not relevant to the code on the branch'. But I'm now getting a different instruction.

**Review by Ibizugbe — COMMENTED** [review]
Hi @calebchris000  and @Tracy-miranja 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a pull request without a  descriptive summary. a descriptive summary should include the changes made in the branch, it gives your repository a professional look😉 kindly edit your summary and write a descriptive summary and submit your PR again. we can't wait to see your awesome project🤗🤗
you can check [this gif](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/images/pull_request_desc.gif) out for a guideline😉
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Tracy-miranja** [comment]
Hi, I have made the changes and my dashboard has not been updated, can you
do the review

File: README.md [hunk 1]
```diff
@@ -37,7 +62,43 @@
...
+
+```git clone https://github.com/calebchris000/Microverse-Potfolio-Project.git ```
+
+
+### Install
+
+Install this project with:
+
+```sudo apt install code ```
+
```
Comment (christianbiring1) [inline README.md]: - [ ] Kindly make sure you have provided the correct commands to make your project more professional.
Hint: You may consider saying:
```sh
To navigate through the cloned repos:
cd <directory_name>
Open with and IDE: e.g VSCode: code.
```

File: README.md [hunk 2]
```diff
@@ -37,7 +62,43 @@
...
+### Usage
+
+To run the project, execute the following command:
+
+**Open a browser and run the index.html file.**
+
+### Run tests
+
+You don't need to run any command.
+
```
Comment (christianbiring1) [inline README.md]: - [ ] I think for testing, the linters were used to achieve this. YOu may consider providing the command to install them and test as well

File: index.html [hunk 3]
```diff
@@ -1,262 +1,359 @@
...
+          <form action="https://formspree.io/f/mlekdbdj" method="post">
+            <ul>
+              <li>
+                <label for="name"></label>
+                <input
+                  type="text"
+                  id="name"
+                  name="user_name"
+                  placeholder="Yeremias"
+                  required
```
Comment (christianbiring1) [inline index.html:325]: - [ ] Kindly you may consider adding some HTML validation to your name field where it says It is supposed to have a `maxlength = 30`

File: styles.css [hunk 4]
```diff
@@ -5,15 +5,27 @@ body {
 .container {
   margin: 0 auto;
   width: 315px;
-  height: 4810px;
+  height: 5210px;
```
Comment (christianbiring1) [inline styles.css]: - [ ] Kindly make sure your page height is limited where the content is limited.

Your design :point_down: 
image

Comment (christianbiring1) [reply styles.css]: - [ ] Also make sure your button has the `hover`, `active`, `disable` state as it is stated in the template
image

File: styles.css [hunk 5]
```diff
@@ -301,11 +313,78 @@ li {
...
+  justify-content: center;
+  margin-left: 60px;
+  border: 2px solid grey;
+  padding: 12px 16px;
+}
+
+button:hover {
+  color: #00f;
+  background-color: #000;
+}
```
Comment (christianbiring1) [inline styles.css]: - [ ] Also make sure your button has the `hover`, `active`, `disable` state as it is stated in the template
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @calebchris000 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] The from spree service is used correctly :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by calebchris000** [comment]
Hello @christianbiring1, I would like for you to re-review my code again.

File: styles.css [hunk 6]
```diff
@@ -5,15 +5,27 @@ body {
 .container {
   margin: 0 auto;
   width: 315px;
-  height: 4810px;
+  height: 5010px;
```
Comment (binhussen) [inline styles.css]: - [Required] Great Job for implementing your previous reviewer suggestions 👍👌, I think you added unnecessary height here, please remove the `height: 5010px;` to stick the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) 👌👍
- [Optional] Kindly, you didn't add the background image, please add the background image to stick to  [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)  👌👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||


😎️

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @calebchris000 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @calebchris000    , I'm @MaganaAsiati , your reviewer for this PR  :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


clap

## Highlights
- descriptive README file  :heavy_check_mark:👨🏻‍💻
- Your commits should start with a capital letter :heavy_check_mark:👨🏻‍💻


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- [ ] kindly  Add a meaningful description in your pull request in the future which give more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Cheers and Happy coding!👏👏👏

**Pull request closed without merging**